README.md

```diff
@@ -82,14 +82,29 @@ const client = new ampDynamicContent.ContentClient({
 
 If you need to support old browsers a legacy version of the bundle is provided, however we strongly recommend using a tool like [babel](https://babeljs.io/) in your project to compile the sdk to your exact browser requirements.
 
-### Fetch content
+### Configuration options
+
+| Option             | Description                                                                                             |
+| ------------------ | ------------------------------------------------------------------------------------------------------- |
+| account            | Required (see Note below): Account to retrieve content from Content Delivery 1 API                      |
+| hubName            | Required (see Note below): hubName to retrieve content from Content Delivery 2 API                      |
```

Comment:
`account` and `hubName` aren't required to make a request if you're using a `stagingEnvironment`. if someone was using this to make a Visualisation they wouldn't ever need to set these. I understand that `account` already is required and you're using `hubName` as a flag for using cdv2. As solely integrating using a `stangingEnvironment` probably isn't that common and we should probably be discouraging it, I'm fine with them being required.

Reply:
Previously if you don't set an `account`  option you will get the error `'Parameter "config" must contain a valid "account" name'`,and I ahve extended the check to look for `hubName` in this PR so if you didn't supply either `account` or `hubName` you will get a similar error.
However I do agree with you, these parameters aren't technically required at the API level, so maybe we should relax the checks in the SDK. Maybe this is something to look at in the future?

Reply:
Yup agree